src/assets/locales/fr.json

```diff
@@ -0,0 +1,710 @@
+    },
+    "attackModifiers": {
+      ".": "Paquet de Modificateur d'Attaque",
+      "hint": "Piocher et éditer le paquet de Modificateur d'Attaque"
+    },
+    "password": {
+      ".": "Mot de passe"
+    },
+    "save": "Sauvegarder les permissions"
+  },
+  "datamanagement": {
+    ".": "Gestion des données",
+    "spoilers": "Gérer les spoilers",
+    "editionDataUrls": "Gérer les données des éditions",
+    "addEditionDataUrl": "Ajouter une URL de donnée d'édition",
+    "restoreEditionDataUrls": "Remettre par défaut",
+    "addSpoiler": "Ajouter un spoiler",
+    "removeAllSpoilers": "Tout supprimer",
+    "importGame": "Importer une partie depuis un fichier",
+    "exportGame": "Exporter la partie vers un fichier",
+    "importSettings": "Importer les paramètres depuis un fichier",
+    "exportSettings": "Exporter les paramètres vers un fichier",
+    "resetGame": "Redémarrer la partie",
+    "resetSettings": "Remettre les paramètres par défaut"
+  },
+  "hints": {
+    "addCharacters": "%character.add%",
+    "setScenario": "%scenario%",
+    "addMonsters": "%monster.add%",
+    "addMonsterEntities": "Utilisez les boutons \"+\" pour générer des monstres.",
+    "draw": "Cliquez sur \"%round.draw%\"",
+    "draw-short": "Définissez l'initiative, puis cliquez sur \"%round.draw%\"",
+    "draw-full": "Choisissez les cartes de capacités des personnages, définissez l'initiative, puis cliquez sur \"%round.draw%\".",
+    "active": "Jouez vos tours",
+    "active-full": "Cliquez sur l'image de chaque figurine après qu'elle ait joué son tour",
+    "next": "Cliquez sur \"%round.next%\""
+  },
+  "about": {
+    ".": "À propos",
+    "ip": "Gloomhaven and all related properties, images and text are owned by {0}.",
+    "license": "Licensed under {0}.",
+    "sourceCode": "Source Code",
+    "support": "Support the dev",
+    "forceUpdate": "Force Update"
+  },
+  "help": "Aide",
+  "update": "Mettre à jour - Recharger",
+  "and": "et",
+  "or": "ou",
+  "cancel": "Annuler",
+  "game": {
+    "action": {
+      "area": "Effets de Zone",
+      "attack": "Attaque",
+      "card": "Carte",
+      "condition": "Condition",
+      "custom": "Custom",
+      "element": "Élément",
+      "elementHalf": "Moitié d'Élément",
+      "fly": "Vol",
+      "heal": "Soin",
+      "jump": "Saut",
+      "loot": "Pillage",
+      "monsterType": "Type de Monstre",
+      "move": "Déplacement",
+      "pierce": "Perforation",
+      "pull": "Traction",
+      "push": "Poussée",
+      "range": "Portée",
+      "refreshItem": "Rafraîchir un objet",
+      "retaliate": "Riposte",
+      "shield": "Bouclier",
+      "special": "Spécial",
+      "specialTarget": "Cible spéciale",
+      "summon": "Invoque",
+      "swing": "Swing",
+      "target": "Cible",
+      "targets": "Cibles",
+      "teleport": "Téléporte",
+      "trigger": "Déclenchement"
+    },
+    "specialTarget": {
+      "self": "Sur lui-même",
+      "self2": "Sur elle-même",
```

Comment:
`self2` should be removed for a single value like `Sur lui-/elle-même`. Besides `selfFemale` is used in data and both label are missing in other languages.